docs/_templates/autosummary/class.rst

```diff
@@ -11,6 +11,7 @@
    .. rubric:: {{ _('Methods') }}
 
    .. autosummary::
```

Comment:
Such a huge change for 2 lines of code changed :) Looks fantastic. Thanks for adding this!

Reply:
:)